x/staking/keeper/keeper.go

```diff
@@ -98,48 +97,18 @@ func (k Keeper) SetLastTotalPower(ctx sdk.Context, power sdk.Int) {
 	store.Set(types.LastTotalPowerKey, bz)
 }
 
-// GetValidatorUpdate returns the ABCI validator power update for the current block
-// by the consensus address.
-func (k Keeper) GetValidatorUpdate(ctx sdk.Context, consAddr sdk.ConsAddress) (abci.ValidatorUpdate, bool) {
-	store := prefix.NewStore(ctx.TransientStore(k.transientKey), types.ValidatorUpdatesKey)
-	bz := store.Get(consAddr.Bytes())
-	if len(bz) == 0 {
-		return abci.ValidatorUpdate{}, false
-	}
-
-	var valUpdate abci.ValidatorUpdate
-	k.cdc.MustUnmarshal(bz, &valUpdate)
-	return valUpdate, true
-}
-
-// HasValidatorUpdate returns true if there is a power update for the given validator
-// within the last block.
-func (k Keeper) HasValidatorUpdate(ctx sdk.Context, consAddr sdk.ConsAddress) bool {
```

Comment:
Was this dead code?

Reply:
Yes. I couldn't find anything within SDK or IS calling it.